db/db_iter.cc

```diff
@@ -764,6 +765,7 @@ bool DBIter::ReverseToBackward() {
         saved_key_.GetUserKey(), kMaxSequenceNumber, kValueTypeForSeek));
     if (!expect_total_order_inner_iter() || auto_readahead_size_) {
       iter_.SeekForPrev(last_key.GetInternalKey());
+      // auto_readahead_size_ = false;
```

Comment:
Temp change to be removed?

Reply:
yes. I was trying to optimize backward scan.